pkg/asset/agent/image/agentpxefiles.go

```diff
@@ -118,6 +120,13 @@ func (a *AgentPXEFiles) PersistToFile(directory string) error {
 		return err
 	}
 
+	if a.pxeBaseURL != "" {
+		err = a.createiPXEScript(a.cpuArch, a.pxeBaseURL, pxeAssetsFullPath)
```

Comment:
It probably wouldn't hurt to strip any trailing `/`s from the URL the user gives us, so we don't double up on them.